### PR TITLE
Rc ourteamhmtl

### PR DESCRIPTION
**Pull request by RachellCalhoun**
I fixed up basic html page, views, help text in forms and css.

### CODE REVIEW

**Comment by RachellCalhoun** [comment]
ourteam1
ourteam2

**Comment by RachellCalhoun** [comment]
@jinpark can you have a look?

File: content/templates/content/our_team.html [hunk 1]
```diff
@@ -2,14 +2,44 @@
...
+      <img class="center-block img-responsive ethicsimage" src="{{team.img.url}}">
+    </div>
+    </div>
+    {% else %}
+   <!-- if no img center the title in the first row -->
+    <div class="col-md-12">
+        <h1 class="childrenstitle text-center text-uppercase">{{team.text|safe}}</h1>
+    </div>
+</div>
+{% endif %}
```
Comment (jinpark) [inline content/templates/content/our_team.html]: can you fix a lot of the indentation?

**Comment by RachellCalhoun** [comment]
done. @jinpark can merge now?

File: content/templates/content/our_team.html [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
-    {% endfor %}
-
-{% endblock %}
\ No newline at end of file
+{% for team in ourteams %}
+{% if forloop.first %}
+<div class="row">
+    <div class="col-md-5 col-md-push-7">
+        {% if team.text %}
+        <h3 class="center-block ourteamtext">{{ team.text|safe}}</h3>
```
Comment (jinpark) [inline content/templates/content/our_team.html]: yay for legibility!

File: content/templates/content/our_team.html [hunk 3]
```diff
@@ -1,15 +1,41 @@
...
+<div class="row">
+    <div class="col-md-12">
+        <h1 class="ethicstitle text-uppercase text-center">Our Peru Team
+        Members:</h1>
+        <div class="teamlist">
+            <h4>{{ team.us_team | linebreaks }}</h4>
+        </div>
+        <h1 class=" text-uppercase text-center">Our US Team Members:</h1>
+        <div class="teamlist">
+            <h4>{{ team.peru_team | linebreaks }}</h4>
```
Comment (jinpark) [inline content/templates/content/our_team.html:32]: yay for spaces!

File: content/templates/content/our_team.html [hunk 4]
```diff
@@ -1,15 +1,41 @@
...
+    <div class="col-md-7 col-md-pull-5"><img class=
+    "center-block img-responsive ethicsimage" src="{{team.img.url}}"></div>
+</div>
+{% else %} <!-- if no img center the title in the first row -->
+<div class="col-md-12">
+    <h1 class="childrenstitle text-center text-uppercase">
+    {{team.text|safe}}</h1>
+</div>
+{% endif %}
+{% endif %}
```
Comment (jinpark) [inline content/templates/content/our_team.html:23]: whats this second `endif` from?

File: content/templates/content/our_team.html [hunk 5]
```diff
@@ -1,15 +1,41 @@
...
\ No newline at end of file
+{% for team in ourteams %}
+{% if forloop.first %}
+<div class="row">
+    <div class="col-md-5 col-md-push-7">
+        {% if team.text %}
+        <h3 class="center-block ourteamtext">{{ team.text|safe}}</h3>
+        {% endif %}
+    </div>{% if team.img %} <!--  if image 4, 8 col -->
+    <div class="col-md-7 col-md-pull-5"><img class=
```
Comment (jinpark) [inline content/templates/content/our_team.html]: can you one line this?

File: content/forms.py [hunk 6]
```diff
@@ -4,8 +4,8 @@
 
 
 class OurTeamForm(forms.ModelForm):
```
Comment (jinpark) [inline content/forms.py:6]: is this used anywhere?

Comment (RachellCalhoun) [inline content/forms.py:6]: This was part of jiwon's code. I think its for the admin page entry of team names.

File: content/templates/content/our_team.html (hunk 4 above)
Comment (RachellCalhoun) [inline content/templates/content/our_team.html:23]: first is ending 'if team.img', second one is ending  'if forloop.first'.

Comment (jinpark) [inline content/templates/content/our_team.html:23]: Cool, then can you fix the indentation for this as well? also put the `if` on https://github.com/djangogirlscodecamp/lightandleadership/pull/39/files#diff-e938681893abc747882aab0ef97b9e75R11 in another line

**Comment by RachellCalhoun** [comment]
i think i fixed it. check it and let me know. thanks

**Pull request merged**